inlong-sort/sort-flink/v1.13/sort-connectors/base/src/main/resources/META-INF/services/org.apache.flink.table.factories.Factory

```diff
@@ -13,5 +13,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-org.apache.inlong.sort.base.dirty.sink.log.LogDirtySinkFactory
+org.apache.inlong.sort.base.dirty.sink.logset.LogDirtySinkFactory
```

Comment:
Makesure it is right?

Reply:
done

Reply:
It is recommended that the names of subsequent files not conflict with the gitignore contents.